src/sacn/source_state.c

```diff
@@ -730,3 +733,10 @@ void remove_from_source_netints(SacnSource* source, const EtcPalMcastNetintId* i
       remove_sacn_source_netint(source, netint_index);
   }
 }
+
+#else  // SACN_SOURCE_ENABLED
+#ifdef _MSC_VER
+// Disable "nonstandard extension used" warning
```

Comment:
We shouldn't be ignoring "nonstandard extension used" warnings - that is the compiler telling you that you are doing something that might not be allowed by the C standard. I'm assuming it's complaining about having an effectively empty source file. If no functions in source_state.c are called when SACN_SOURCE_ENABLED=0, we can just leave it as-is without guarding it with #ifdefs; the linker will automatically remove the code from this file if it's not used by anything else.

Reply:
https://github.com/ETCLabs/sACN/pull/49/commits/6755f35cff4856a60753c813fbe8e7ede409303f https://github.com/ETCLabs/sACN/pull/49/commits/8b33cfb551932af29a0e42251b81f42b2817afe9 https://github.com/ETCLabs/sACN/pull/49/commits/aa8b0328a3d56468ffe5122d5b359229022ee887

Reply:
I'm assuming the linker will also remove them if the only functions that call them are also unused functions, and so forth.